README.md

```diff
@@ -52,4 +52,21 @@ can be overriden by the environment variable `DOCKER_GATEWAY_HOST`:
 ```bash
 EXPORT DOCKER_GATEWAY_HOST=`docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}'
 npm start
+```
+
+## Debugging locally
+
+To save the arguments (`config`, `scdata`, etc) to a task function, specify DEBUG_STEP and DEBUG_PATH:
+
+```bash
+# e.g. DEBUG_STEP=dataIntegration
```

Comment:
I suppose the comment here could is a good place to list all available tasks, i.e. adding:
# available tasks are:
`cellSizeDistribution` `mitochondrialContent` `classifier` `numGenesVsNumUmis` `doubletScores` `dataIntegration` `configureEmbedding`

Reply:
Instead of explicitly listing all the steps I added link to where steps are defined. Might make it easier to for future updates/name changes. Thanks!

Reply:
Instead of explicitly listing all the steps I added link to where steps are defined. Might make it easier to for future updates/name changes. Thanks!